project/simpson0114/README.rst

```diff
@@ -0,0 +1,66 @@
+==================================================
+Minimum Spanning Tree Algorithm
+==================================================
+
+Basic Information
+=================
+**Minimum Spanning Tree(MST)** is a significant graph algorithm structure. In a connected, edge-weighted undirected graph, 
+a minimum spanning tree is a subset of edges that connects all nodes with the smallest total weights and has been well-studied for nearly a century.
+
+Problem to Solve
+=================
+In the project we try to make MST faster and easier
+
+MST is a classic graph algorithm, and it is useful to many problems in real life. However implement a MST algorithm is not a walk in the park, 
+we hope to build a framework that user just have to decide the input graph vertex and edge number, and the range of the edges weights.
+
+There are two famous Minimum spanning tree (MST) algorithms, which are Prim's algorithm and Kruskal's algorithm respectively. We will implement 
+this two MST algorithms, and hope it can make it easier for beginners to understand this two type of MST algorithms.
+
+Prospective Users
+==================
+People who wants to use MST but have no idea of programming or people who just start to learn the MST.
+
+System Architecture
+====================
+
+.. image:: images/MST_system_architecture.png
+
+API Description
+=================
+
+Graph class includes complete implementation of the Kruskal' algorithm and Prim's algorithm, user only
```

Comment:
It will help your development by a lot to describe the data structure that you want to use for your graph algorithms.  You may find some algorithms are hard to implement if the data structure does not fit.  Performance will also be an issue.